zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/PartitionContext.java

```diff
@@ -76,4 +76,6 @@ public interface PartitionContext {
   ControllableStreamClock getStreamClock();
 
   void setStreamClock(ControllableStreamClock clock);
+
+  int getPartitionCount();
```

Comment:
:wrench: Not sure if we will update it dynamically. So it would be good to document that this is the statically  configured partitionCount.